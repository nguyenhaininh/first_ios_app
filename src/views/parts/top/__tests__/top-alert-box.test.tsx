/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import TopAlertBox from '../top-alert-box';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const fn = jest.fn();
  const tree = renderer.create(
    <TopAlertBox onPress={fn}>
      Test
    </TopAlertBox>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
