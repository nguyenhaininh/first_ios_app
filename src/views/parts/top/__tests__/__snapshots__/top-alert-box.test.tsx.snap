// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#ffcd00",
      "borderWidth": 2,
      "justifyContent": "center",
      "marginHorizontal": 16,
      "marginVertical": 10,
      "minHeight": 51,
      "opacity": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#e62222",
        "fontSize": 11,
        "fontWeight": "bold",
      }
    }
  >
    Test
  </Text>
</View>
`;
